draft-ietf-core-object-security.md

```diff
@@ -1519,6 +1521,46 @@ This section registers the media type 'application/oscore' media type in the "Co
 ~~~~~~~~~~~
 {: artwork-align="center"}
 
+## OSCORE Octet Registry {#oscore-octet}
+
+It is requested that IANA create a new registry entitled "OSCORE Octet". The registry should be created with the Expert Review policy. Guidelines for the experts are provided in {{exp-instr}}.
+
+The columns of the registry are:
+
+* bit position: This indicates the position of the bit in the OSCORE Octet, starting at 0 for the most significant bit. The bit position must be an integer or a range of integers, in the range 0 to 63.
+
+* name: The name is present to make it easier to refer to and discuss the registration entry. The value is not used in the protocol. Names are to be unique in the table.
+
+* description: This contains a brief description of the use of the bit.
+
+* specification: This contains a pointer to the specification defining the entry.
+
+The initial contents of the registry can be found in {{table-flagbyte}}. The specification column for all rows in that table should be this document. Additionally, the entry with Bit Position of 0 is to be marked as 'Reserved'.
+
```

Comment:
Put bit position 0 into the table below as well.  Do we need to say why we are reserving it so that we can remember this is three years when it gets to the point that it will need to be used?